src/decode.h

```diff
@@ -778,6 +779,126 @@ void CaptureStatsSetup(ThreadVars *tv, CaptureStats *s);
         }                                           \
     } while(0)
 
+/**
+ *  \brief Initialize a packet structure for use.
+ */
+#define PACKET_INITIALIZE(p)                                                                       \
```

Comment:
https://github.com/OISF/suricata/pull/7972#discussion_r990642443

Reply:
@jlucovsky leaving this back to you ;-)

Reply:
Will update